tests/ga/test_exthandlers.py

```diff
@@ -231,6 +231,10 @@ def setUp(self):
         self.mock_get_log_dir = patch("azurelinuxagent.ga.exthandlers.ExtHandlerInstance.get_log_dir", lambda *_: self.log_dir)
         self.mock_get_log_dir.start()
 
+        mock_sleep = time.sleep
```

Comment:
Can you rebase to develop and remove the `SLEEP` mocking in the header of this file, if we're mocking it this way now? Thanks.

Reply:
Nevermind, wrong file.